config.toml

```
@@ -114,3 +114,7 @@ sphinxopts = [
     '-D latex_elements.inputenc=',
     '-D latex_elements.fontenc=\\usepackage{xeCJK}',
 ]
+
+[languages.el]
+name = "Greek"
+repo_name = "python-docs-gr"
```

Please also update to add the new `translated_name` key.
```suggestion
repo_name = "python-docs-el"
```